mypy.ini

```
@@ -0,0 +1,43 @@
+# Global options
+
+[mypy]
+ignore_missing_imports = True
+disallow_untyped_defs = True
+show_error_codes = True
+
+#########################
+
+[mypy-dash.development.*]
+ignore_errors=True
+
+[mypy-dash.testing.*]
+ignore_errors=True
+
+[mypy-dash.dash]
+ignore_errors=True
+
+[mypy-dash.resources]
+ignore_errors=True
+
+[mypy-dash._callback_context]
+ignore_errors=True
+
+[mypy-dash._validate]
+ignore_errors=True
+
+[mypy-dash.dependencies]
+ignore_errors=True
+
+#########################
+
+[mypy-dash.html.*]
+ignore_errors=True
+
+[mypy-dash.dcc.*]
+ignore_errors=True
+
+[mypy-dash.dash_table.*]
+ignore_errors=True
+
+[mypy-dash.long_callback.managers.*]
+ignore_errors=True
```

The list of `ignore errors` will be shortened if PR is taken further.